frontend/src/lib/apiClient.js

```diff
@@ -14,6 +14,9 @@ const getRecipes = (search) => {
     .then(results => results.data);
 };
 
+const getRecipesAdmin = () => apiClient.get('api/v1/recipe')
+  .then(results => results.data);
```

Comment:
I don't think you need a separate function. `getRecipes` should do what you want, just don't pass in anything.